bin/test-run

```diff
@@ -83,15 +83,19 @@ function run_upgrade_test() {
 }
 
 function run_helm_test() {
+    kubectl --context=$k8s_context create ns $tiller_namespace
+    kubectl --context=$k8s_context create clusterrolebinding tiller-cluster-admin --clusterrole=cluster-admin --serviceaccount=${tiller_namespace}:default
```

Comment:
maybe this is better:
`tiller-cluster-admin` => `${tiller_namespace}:tiller-cluster-admin`